distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -4,6 +4,7 @@ import Standard.Base.Data.Filter_Condition as Filter_Condition_Module
 import Standard.Base.Data.Time.Errors.Date_Time_Format_Parse_Error
 import Standard.Base.Data.Vector.Builder
 import Standard.Base.Errors.Common.Additional_Warnings
+import Standard.Base.Errors.Common.Floating_Point_Equality
```

Comment:
Is this needed? Looks like only change to this file.

Reply:
`Floating_Point_Equality` is used in this file, and was previously imported by `from Standard.Table.Errors import all`, but not anymore because it has moved to Common.